kong/plugins/zipkin/reporter.lua

```diff
@@ -71,6 +69,10 @@ function zipkin_reporter_methods:report(span)
     zipkin_tags = nil
   end
 
+  local localEndpoint = {
+    serviceName = self.local_service_name
+  }
+
```

Comment:
instead of defineing a local here, can't we just define it inline below in the `zipkin_span`;

Change
```lua
    localEndpoint = localEndpoint,
```
to
```lua
    localEndpoint = { serviceName = self.local_service_name },
```

Reply:
Fixed